workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Readers/IReader.cs

```diff
@@ -7,15 +7,21 @@ namespace Improbable.Gdk.Core.MonoBehaviours
     ///     callbacks for updates, events and commands for component instances.
     /// </summary>
     /// <typeparam name="TSpatialComponentData">The data type for the SpatialOS component.</typeparam>
-    /// <typeparam name="TComponentUpdate">The update type for the SpatialOS component.</typeparam>
-    public interface IReader<TSpatialComponentData, TComponentUpdate>
+    public interface IReader<TSpatialComponentData>
```

Comment:
I should've asked this in a previous PR, but is anything actually going to refer to the readers via this interface?

Reply:
Yes, when the users use a `SomeComponent.Reader` interface they will be allowed to access only the methods declared here.

Reply:
Ie they'll be referring to it as SomeComponent.Reader, not IReader<Stuff>. But that's fine, no need to redeclare similar interfaces when you can just put it in a generic supertype, and also lets users see the similarities/relations I guess.